bin/sparqlalgebrajs.ts

```diff
@@ -26,5 +27,5 @@ if (args.r || args.reverse)
 }
 else
 {
-    console.log(JSON.stringify(Util.objectify(translate(args._[0], { quads: args.q })), null, 2));
-}
\ No newline at end of file
+    console.log(JSON.stringify(Util.objectify(translate(args._[0], { quads: args.q, sparqlStar: !args.s })), null, 2));
```

Comment:
Probably also want to include the `sparqlStar` option in the `toSparql` call above in case you don't know where the input is coming from.

Reply:
Ah of course, I missed that one...